### DISCUSSION TITLE
关于共享至 zstu.wiki 的许可

### DISCUSSION THREAD

**Discussion by flottant** [chen2438/zstu-study#42] [Q&A] [answered]
是否同意搬运部分内容至 https://zstu.wiki ？这是一个新建的免费资源网站，目前个人整理了大约50门课程，预期在期中考试前整理完毕，最晚不超过期末考试时。对于您独创的笔记，考虑使用知识共享协议进行转载。

**Comment by chen2438** [comment, marked as answer]
可以的

**Comment by flottant** [comment]
网站已经搭建完成了。
